src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+@import url(https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic);
+
+*,
+*::before,
+*::after {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --text-size: 12px;
+  --active-link: #00acdc;
+  --header-height: 60px;
+}
+
+html,
+body {
+  overflow-x: hidden;
+  font-size: 62.5%;
+}
+
+.page__header {
+  width: 100%;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item:not(:first-child) {
+  height: var(--header-height);
+  margin-left: 20px;
+  display: flex;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  position: relative;
+  font-size: var(--text-size);
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  line-height: var(--header-height);
+  text-decoration: none;
+  color: inherit;
+  height: 100%;
+  display: inline-block;
+}
+
+.nav__item:hover > .nav__link {
```

Comment:
I don't think we need to be that specific with those selectors, just put the hover on concrete element 😄

Reply:
I thought that the more specific the better...Ofcourse I will fix that :)